src/App/Pages/Vault/ViewPage.xaml

```diff
@@ -571,6 +571,7 @@
                                                 IsVisible="{Binding IsLinkedType}" />
                                             <controls:IconLabel
                                                 Text="{Binding ValueText, Mode=OneWay}"
+                                                AutomationProperties.Name="{Binding ValueAccessibilityText, Mode=OneWay}"
```

Comment:
We need to add here `AutomationProperties.IsInAccessibleTree="true"` [(docs)](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/accessibility/automation-properties#automationpropertiesisinaccessibletree) as well so we don't have any weird issues on some devices